app/containers/ViewQuestion/saga.js

```diff
@@ -52,8 +65,26 @@ export function* getQuestionDataWorker(res) {
 
 export function* postCommentWorker(res) {
   try {
+    let questionData = yield select(selectQuestionData());
     const eosService = yield select(selectEos);
 
+    const profileInfo = yield call(() => getProfileInfo(res.user, eosService));
+
+    if (!profileInfo) {
+      yield put(showLoginModal());
+      throw new Error('Not authorized');
+    }
+
+    yield call(() =>
+      postCommentValidator(
```

Comment:
`postCommentValidator` should not raise exception. It needs to return false if validation fails and  postCommentWorker `return` if false.